services/Directory/data/FilterList.json

```diff
@@ -17371,5 +17371,16 @@
     "issuesUrl": "https://github.com/DandelionSprout/adfilt/issues",
     "licenseId": 35,
     "name": "Anti-'Stock vector art on copypaste backgrounds'"
+  },
+  {
+    "description": "Filter of malicious web resources (phishing, online fraud, malware, etc.) targeting Ukrainian citizens",
+    "donateUrl": "https://www.paypal.com/donate/?hosted_button_id=AQ9BB34WHFADC",
+    "homeUrl": "https://github.com/braveinnovators/ukrainian-security-filter",
+    "id": 2669,
+    "issuesUrl": "https://github.com/braveinnovators/ukrainian-security-filter/issues",
+    "licenseId": 4,
+    "name": "Ukrainian Security Filter",
+    "syntaxId": 3,
```

Comment:
Syntax should be declared in [FilterListSyntax.json](https://github.com/collinbarrett/FilterLists/blob/main/services/Directory/data/FilterListSyntax.json).
Thanks

Reply:
Fix it